src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/DotNetXamarinProject.cs

```diff
@@ -26,9 +26,9 @@ protected DotNetXamarinProject (string debugConfigurationName = "Debug", string
 			SetProperty ("AssemblyName", () => AssemblyName ?? ProjectName);
 
 			if (Builder.UseDotNet) {
-				SetProperty ("TargetFramework", "net8.0-android");
-				SetProperty ("EnableDefaultItems", "false");
-				SetProperty ("AppendTargetFrameworkToOutputPath", "false");
+				TargetFramework = "net8.0-android";
+				EnableDefaultItems = false;
+				AppendTargetFrameworkToOutputPath = false;
 			} else {
```

Comment:
Shouldn't we just delete this entire `else` block?  We're not supporting Classic, right?

Reply:
I would rather try to address this type of clean up in a future PR.